lib/sqlalchemy/dialects/mssql/__init__.py

```diff
@@ -42,6 +42,7 @@
 from .base import VARBINARY
 from .base import VARCHAR
 from .base import XML
+from .base import TRY_CAST
```

Comment:
this should be lowercase then, right?  did you run the tests?

Reply:
I had not, tbh. Taking a look at https://github.com/sqlalchemy/sqlalchemy/blob/master/README.unittests.rst

Reply:
I was hitting https://github.com/astropy/astropy/issues/8555. In case anyone else faces this, just do `pip install pytest-xdir`